packages/studio/src/components/VersionNumber.tsx

```diff
@@ -0,0 +1,40 @@
+import { useCallback } from "react";
+import { version } from "../../package.json";
+import { ReactComponent as Info } from "../icons/info.svg";
+import ButtonWithModal, { renderModalFunction } from "./common/ButtonWithModal";
+import Modal from "./common/Modal";
+
+const infoSVG = <Info className="h-7" />;
+const body = (
+  <div>
+    <p className="font-bold mb-4"> Studio Info</p>
+    Studio Verison: {version}
+  </div>
+);
+
+/**
+ * Renders the current version number
+ */
+export default function VersionNumber() {
+  const renderModal: renderModalFunction = useCallback(
```

Comment:
we can move this outside the component body!

Reply:
oh, so true